homeassistant/components/swiss_public_transport/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: done
+    comment: >
+      Polling interval is set to support one connection.
+      There is a rate limit at 10000 calls per day.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: No events implemented
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable:
+    status: done
+    comment: Offloaded to coordinator
+  entity-unavailable:
+    status: done
+    comment: Offloaded to coordinator
+  action-exceptions: done
+  reauthentication-flow:
+    status: exempt
+    comment: No authentication needed
+  parallel-updates: done
+  test-coverage: done
```

Comment:
This one isn't done. This rule requires all modules to have a test coverage above 95%

![CleanShot 2024-11-26 at 21 42 41@2x](https://github.com/user-attachments/assets/2631da58-b852-4027-8643-77aa6008ac9c)